### PR TITLE
chore(skip-release): Prepare for 1.20.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore(skip-release): Prepare for 1.20.0

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -11,6 +11,14 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.20.0</b></p>
+    <ul>
+      <li>Support for Renarde Quarkus application (URL codelens in Java file, uri/uriabs support in Qute template)</li>
+      <li>Support for Qute fragments</li>
+      <li>Support Quarkus3 for Qute</li>
+      <li>Completion context aware for snippets</li>
```
Comment (fbricon) [inline src/main/resources/META-INF/plugin.xml:19]: snippets are not rendered correctly so I'm not sure it's worth mentioning

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -11,6 +11,14 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.20.0</b></p>
```
[the remark this answers is not in the document]
Comment (fbricon) [reply src/main/resources/META-INF/plugin.xml:14]: either way there should be a link to the full changelog

[the remark this answers is not in the document]
Comment (fbricon) [reply src/main/resources/META-INF/plugin.xml:14]: See how Kotlin plugin does it: 

Screenshot 2023-06-07 at 12 19 48

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (angelozerr) [reply src/main/resources/META-INF/plugin.xml]: removed

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (angelozerr) [reply src/main/resources/META-INF/plugin.xml]: I updated link for millestones which was updated correctly

[the remark this answers is not in the document]
Comment (fbricon) [reply src/main/resources/META-INF/plugin.xml]: on 2nd thought, I think a "Learn more in the changelog" link will be more obvious

[the remark this answers is not in the document]
Comment (angelozerr) [reply src/main/resources/META-INF/plugin.xml]: I updated html structure with h3 and a link like kotlin ij plugin

**Review by fbricon — APPROVED** [review]

**Pull request merged**